opencensus/proto/trace/trace.proto

```diff
@@ -62,10 +66,11 @@ message Span {
   // is the time when the server's application handler starts running.
   google.protobuf.Timestamp start_time = 5;
 
-  // The end time of the span. On the client side, this is the time kept by
-  // the local machine where the span execution ends. On the server side, this
-  // is the time when the server application handler stops running.
-  google.protobuf.Timestamp end_time = 6;
+  // The duration of the span. On the client side, this is the time kept by
+  // the local machine from start_time until the span execution ends. On the server
+  // side, this is the duration between the start_time and when the server
+  // application handler stops running.
+  google.protobuf.Duration duration = 14;
```

Comment:
Why do we need this? end_time - start_time must be equal to this.

Reply:
To avoid questions of wall clock changes https://github.com/census-instrumentation/opencensus-proto/pull/49#issuecomment-370283102